0xblock-co/bleep-web
# few changes

### Describe your changes

### Issue ticket number and link

### Type of change


### How to Test

### Checklist before requesting a review

#### General

- [ ] I merged the current development branch (before testing)
- [ ] I have double-checked target branch
- [ ] I joined a screenshot of the app before and after the fix.

#### API Checklist before requesting a review

- [ ] added/updated API testing endpoints and documentation

#### Frontend Checklist before requesting a review


#### Mobile Apps Checklist before requesting a review

- [ ] tested all links in project relevant devices

<details>
  <summary>Notes</summary>
  
- Add required [labels](https://github.com/organizations/0xblock-co/settings/repository-defaults label)
- Follow [the code review prcoess](https://0xblock.atlassian.net/wiki/spaces/0XBLOCK/pages/52133889/Web+Code+Review+Process)
- Follow [the code guidelines](https://0xblock.atlassian.net/wiki/spaces/0XBLOCK/pages/52199425/Coding+Guidelines)
- If a PR should be merged before this, add a note.
- Please self-review your code before submitting the PR.
- Add localised string to `en_GB.json`
</details>
